src/erk/artifacts/sync.py

```diff
@@ -0,0 +1,111 @@
+"""Artifact synchronization from erk package to project."""
+
+from dataclasses import dataclass
+from pathlib import Path
+
+from erk.artifacts.models import ArtifactState
+from erk.artifacts.staleness import get_current_version
+from erk.artifacts.state import save_artifact_state
+from erk.kits.hooks.installer import install_hooks
+from erk.kits.io.manifest import load_kit_manifest
+from erk.kits.models.artifact import ARTIFACT_TARGET_DIRS, ArtifactType
+from erk.kits.operations.artifact_operations import create_artifact_operations
+from erk_kits import get_kits_dir
+
+
+@dataclass
```

Comment:
make it frozen

Reply:
Resolved via /erk:pr-address at 2025-12-30 05:33 AM PST

_Addressed via `/erk:pr-address` at 2025-12-30 05:33 PST_